packages/editor-sdk/src/components/Widgets/ModuleWidget.tsx

```diff
@@ -19,6 +24,35 @@ declare module '../../types/widget' {
   }
 }
 
+const genSpec = (type: Types, target: any): TSchema => {
```

Comment:
Why not use a library to convert json to JsonSchema?

Reply:
I think this conversion function is easy to implement and easier to maintain  by implementing it yourself.
What do you think?

Reply:
ok.